Auth modal

crist6494 opened the pull request.

crist6494 committed:
feat: create basic modal elements

crist6494 committed:
feat: create modal elements with js actions

crist6494 committed:
feat: create modal with all elements

crist6494 committed:
feat: translate modal form

crist6494 committed:
feat: connect modal form with backed

crist6494 committed:
style: modify and fix some attribute style

crist6494 committed:
feat: improve form properties of the modal

crist6494 committed:
fix: resolve translate issue

crist6494 committed:
fix(web-component): evaluate attributes  on childs

crist6494 committed:
style(eslint): resolve syntax problems

crist6494 committed:
refactor(register-modal): split code into generic modal and register-section

crist6494 committed:
refactor(app-rooter): create a new component

crist6494 committed:
refactor(landing-navbar): add variable to show modal

crist6494 committed:
Merge branch 'betta' into feature/TR-39/Crear-modal-registro

jdomingu98 committed:
fix(register-modal): change link of terms and conditions

jdomingu98 reviewed (changes requested):
Lo que comentamos en la reunion:

- Borde rojo cuando un campo no esta bien relleno
- Opacidad del boton de registro. 2 opciones:
  - Mantener opacidad, pero deshabilitar el boton hasta que esten rellenos los datos. (Validaciones de campos sin rellenar ya no serian necesarios)
  - Quitar opacidad y que tenga un color fijo (el resto igual en principio)
- Falta settear el token en session storage
- Falta redireccion al panel del usuario (perfil)
- Mejorar proceso de validacion del formulario. No esta bien que primero te diga unos errores y que cuando los corrijas te salgan otros en otro lado que antes no te habian dicho nada (ningun formulario de registro funciona así).
- En vez del texto "-OR-", pon "- OR -"
- Añadir padding bottom en los errores de validaciones (esta bastante pegado)

PD: No lo subas todo en un commit y no lo pushees el día del cierre del sprint.

ruzafa8 commented on src/components/app/landing/landing-modal/register-section/register-section.js:
```diff
@@ -0,0 +1,181 @@
+                            [key]: msgError
+                        }
+                    });
+                    if(`${key}` === 'username' || `${key}` === 'checkbox'){
```
¿Es necesario hacer un
```js
`${key}`
```

Podría valer con:
```js
key === 'username' || key === 'checkbox'
```

ruzafa8 replied:
```suggestion
                    if(key === 'username' || key === 'checkbox'){
```

crist6494 replied:
👍🏻

ruzafa8 commented on src/components/common/forms/form-field/form-field.js:
```diff
@@ -0,0 +1,34 @@
+    render() {
+        const error = this.getAttribute('error') || '';
+        const rl = this.getAttribute('rl') || 'input';
+        const text = this.getAttribute('text') || 'text';
```
Valor por defecto: No pondría "text" como tal... Si no ponemos text, queremos que esté vacío, por lo cual pondría cadena vacía:

```suggestion
        const text = this.getAttribute('text') || '';
```

jdomingu98 replied:
Añado a esto:
que es r1? mejor nomenclatura plis. El codigo tiene que ser lo suficientemente legible para que yo, que no he visto nada del codigo, pueda entenderlo todo sin problemas sin tener que estar buscando todo el rato por el codigo

ruzafa8 commented on src/components/app/landing/landing-modal/register-section/register-section.js:
```diff
@@ -0,0 +1,181 @@
+                    hasErrors = true;
+                }
+            });
+            return hasErrors ? false : true;
```
```suggestion
            return !hasErrors;
```

ruzafa8 commented on src/components/common/forms/form-field/form-field.js:
```diff
@@ -0,0 +1,34 @@
+        const error = this.getAttribute('error') || '';
+        const rl = this.getAttribute('rl') || 'input';
+        const text = this.getAttribute('text') || 'text';
+        const type = this.getAttribute('type') || 'type';
```
```suggestion
        const type = this.getAttribute('type') || 'text';
```

Este valor lo enchufamos en el atributo type de un input. Si miras la documentación, https://www.w3schools.com/html/html_form_input_types.asp, no se le puede pasar "type" como tal, por defecto si no le pasamos nada es "text", así podríamos respetarlo.

ruzafa8 commented on src/components/common/forms/form-field/form-field.js:
```diff
@@ -0,0 +1,34 @@
+        const rl = this.getAttribute('rl') || 'input';
+        const text = this.getAttribute('text') || 'text';
+        const type = this.getAttribute('type') || 'type';
+        const placeholder = this.getAttribute('placeholder') || 'Enter your email or username';
```
1. Si no ponemos placeholder, lo ideal sería que no se mostrara placeholder.
2. Si le dejasemos un texto, debería de ser un texto traducible (Literal de traducción)

Yo prefiero que sea cadena vacía:
```suggestion
        const placeholder = this.getAttribute('placeholder') || '';
```

ruzafa8 commented on src/components/common/forms/form-field/form-field.js:
```diff
@@ -0,0 +1,34 @@
+
+        return `
+            <div class="form-group d-flex flex-column mb-2">
+                <label for="${rl}}" class="text-uppercase mb-2 mt-3 text-start">${text}</label>
```
typo!! Cuidado:
```suggestion
                <label for="${rl}" class="text-uppercase mb-2 mt-3 text-start">${text}</label>
```

ruzafa8 commented on src/components/common/texts/modal-span-text.js:
```diff
@@ -0,0 +1,33 @@
+        const mb = this.getAttribute('mb') || '';
+
+        return `
+            <span class="text text-start d-flex justify-content-center" style="margin-top:${mt}; margin-bottom:${mb};">${content}</span>
```
```suggestion
            <span class="text text-start d-flex justify-content-center" style="margin-top:${mt}; margin-bottom:${mb};">{{innerHTML}}</span>
```

ruzafa8 replied:
Yo no tendría este componente!!! Un componente con un span solo... sobra un poco... Pondría directamente el span en el padre

ruzafa8 replied:
Además... no crearía un atributo que sea mb... usaría algo así como className para pasarle toodo el class que necesite, en vez de un style.

crist6494 committed:
fix(translator): change "or" msg sintax

crist6494 committed:
feat(services): add new services

crist6494 committed:
fix(modal-register): improve inputs errors and styles

crist6494 committed:
refactor(webComponent): add conditional to be safer

jdomingu98 commented on lib/services/NavigatorService.js:
```diff
@@ -0,0 +1,21 @@
+    }
+
+    static goToHome() {
+        Router.push('/app');
```
El home no es este, sería "/app/profile/:id", donde ":id" sería el id del usuario

jdomingu98 commented on src/components/app/landing/landing-modal/modal-card.js:
```diff
@@ -0,0 +1,61 @@
+
+import css from './modal-card.css?inline';
+
+
```
espacios de mas

jdomingu98 commented on src/components/app/landing/landing-modal/modal-card.js:
```diff
@@ -0,0 +1,61 @@
+
+
+export default Component(
+    {
```
mal formateado

jdomingu98 commented on src/components/app/landing/landing-modal/modal-card.js:
```diff
@@ -0,0 +1,61 @@
+            });
+        }
+
+        closeEsc = (e) => {
```
parentesis innecesarios

jdomingu98 commented on src/components/app/landing/landing-modal/modal-card.js:
```diff
@@ -0,0 +1,61 @@
+
+            this.subscribe('register-section', 'CLOSE_MODAL', () => this.emit('CLOSE_MODAL'));
+
+            document.addEventListener('keydown', this.closeEsc);
```
no se usa addEventListener, en la libreria ya esta implenmentado subscribe

jdomingu98 commented on src/components/app/landing/landing-modal/modal-card.js:
```diff
@@ -0,0 +1,61 @@
+
+            document.addEventListener('keydown', this.closeEsc);
+            return () => {
+                document.removeEventListener('keydown', this.closeEsc);
```
Lo mismo. Si por un casual @ruzafa8 no implementó el unsuscribe pues que lo añada

jdomingu98 commented on src/components/app/landing/landing-modal/modal-card.js:
```diff
@@ -0,0 +1,61 @@
+
+        render() {
+            return `
+                <div class="overlay"></div>
```
Mu cutre esto. Usa la etiqueta dialog para los modales (Investigalo)

jdomingu98 commented on src/components/app/landing/landing-modal/register-section/register-section.js:
```diff
@@ -0,0 +1,245 @@
+
+            Object.entries(data).forEach(([key, value]) => {
+                if (value === '') {
+                    errors[key] = `${capitalizeFirstLetter(key)} is required`;
```
Mas que hacer una funcion especificamente para cambiar la primera letra, no seria mejor añadirle algo antes.
Ej: "A 'password' is required"

También hay que traducir esto!!

jdomingu98 commented on src/components/app/landing/landing-modal/register-section/register-section.js:
```diff
@@ -0,0 +1,245 @@
+                    hasErrors = true;
+                } else {
+                    if (key === 'username' && !this.validateUsername(username)) {
+                        errors.username = 'Invalid username';
```
Traduccion

jdomingu98 commented on src/components/app/landing/landing-modal/register-section/register-section.js:
```diff
@@ -0,0 +1,245 @@
+                    }
+
+                    if (key === 'email' && !this.validateEmailFormat(email)) {
+                        errors.email = 'Invalid email (name@example.com)';
```
Traduccion (Lo que esta entre parentesis sobra)

jdomingu98 commented on src/components/app/landing/landing-modal/register-section/register-section.js:
```diff
@@ -0,0 +1,245 @@
+                    }
+
+                    if (key === 'password' && !this.validatePassword(password)) {
+                        errors.password = 'Invalid password';
```
Traduccion

jdomingu98 commented on src/components/app/landing/landing-modal/register-section/register-section.js:
```diff
@@ -0,0 +1,245 @@
+                    }
+
+                    if (key === 'confirm_password' && password !== confirm_password) {
+                        errors.confirm_password = 'Passwords do not match';
```
Traduccion

jdomingu98 commented on src/components/app/landing/landing-modal/register-section/register-section.js:
```diff
@@ -0,0 +1,245 @@
+            });
+
+            if (!this.handleCheckboxChange()) {
+                errors.checkbox = 'Please accept the terms to continue.';
```
Traduccion

jdomingu98 commented on src/components/app/landing/landing-modal/register-section/register-section.js:
```diff
@@ -0,0 +1,245 @@
+
+import css from './register-section.css?inline';
+
+function capitalizeFirstLetter(str) {
```
Innecesario

jdomingu98 commented on src/components/app/landing/landing-modal/register-section/register-section.js:
```diff
@@ -0,0 +1,245 @@
+    return str.charAt(0).toUpperCase() + str.slice(1);
+}
+
+
```
espacio de mas

jdomingu98 commented on src/components/app/landing/landing-modal/register-section/register-section.js:
```diff
@@ -0,0 +1,245 @@
+
+
+export default Component(
+    {
```
formato

jdomingu98 commented on src/components/app/landing/landing-modal/register-section/register-section.js:
```diff
@@ -0,0 +1,245 @@
+        }
+
+        validateUsername(username) {
+            const usernameRegex = /^[a-zA-Z0-9_]{3,20}$/;
```
Deberia estar en consts.js

jdomingu98 replied:
Alinear con el backend el regex si no lo esta

jdomingu98 commented on src/components/app/landing/landing-modal/register-section/register-section.js:
```diff
@@ -0,0 +1,245 @@
+        }
+
+        validateEmailFormat(email) {
+            const emailRegex = /^[^\s@]+@[^\s@]+\.[^\s@]+$/;
```
Deberia estar en consts.js

jdomingu98 replied:
Alinear con el backend el regex si no lo esta

jdomingu98 commented on src/components/app/landing/landing-modal/register-section/register-section.js:
```diff
@@ -0,0 +1,245 @@
+        }
+
+        validatePassword(password) {
+            const passwordRegex = /^(?=.*[a-z])(?=.*[A-Z])(?=.*\d)(?=.*[@.#$!%*?&^])[A-Za-z\d@.#$!%*?&]{8,15}$/;
```
Deberia estar en consts.js

jdomingu98 replied:
Alinear con el backend el regex si no lo esta

jdomingu98 commented on src/components/app/landing/landing-navbar.js:
```diff
@@ -36,10 +66,11 @@ class LandingNavBar extends WebComponent {
                     {{ translator.translate("LANDING.NAVBAR.LOGIN") }}
                 </primary-button>
             </nav>
+            ${this.state.showRegister ? '<modal-card id="modal-register"></modal-card>' : ''}
```
Estoy casi seguro que se puede gestionar esto con el dialog. Solo es suscribir el boton a un evento de click y darle al dialog el atributo open

jdomingu98 commented on src/components/common/buttons/primary-button.js:
```diff
@@ -3,7 +3,41 @@ import WebComponent, { Component } from '#WebComponent';
 export default Component ({
-    tagName: 'primary-button'
+    tagName: 'primary-button',
+    styleCSS: `
```
Todo esto donde se usa?

jdomingu98 replied:
el css me refiero

jdomingu98 commented on src/components/common/buttons/primary-button.js:
```diff
@@ -12,14 +46,16 @@ class PrimaryButton extends WebComponent {
         return `
             <button
-                class="btn btn-primary px-0 border-0 fw-semibold text-uppercase rounded-pill ${bootstrap}"
-                style=" width:${width}; height:${height}; background-color: ${color}; letter-spacing: 0.1em; font-size: 16px;"
```
Por que has quitado el letter-spacing y el font-size?

jdomingu98 commented on src/components/common/buttons/primary-button.js:
```diff
@@ -12,14 +46,16 @@ class PrimaryButton extends WebComponent {
         const color = this.getAttribute('color') || BUTTON_DEFAULT_PRIMARY_COLOR;
         const buttonText = this.innerHTML || BUTTON_DEFAULT_MSG;
-        const bootstrap = this.getAttribute('bootstrap') || '';
+        const claseName = this.getAttribute('claseName') || '';
```
className en lugar de claseName (Pendiente de mirar de como lo usas esto)

jdomingu98 commented on src/components/common/forms/box-form-field/box-form-field.css:
```diff
@@ -0,0 +1,35 @@
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    color: #8D8DDA;
```
Usa las variables globales

jdomingu98 commented on src/components/common/forms/box-form-field/box-form-field.css:
```diff
@@ -0,0 +1,35 @@
+}
+
+.terms-group .terms-link {
+    color: #a8c0f0; 
```
Colores random no. Creo que este color es el del tick del checkbox, asi que usa el color verde de las variables globales

jdomingu98 replied:
Tambien hay un color azul global para el trozo de texto

jdomingu98 commented on src/components/common/forms/form-field/form-field.css:
```diff
@@ -0,0 +1,81 @@
+
+.form-group input:focus {
+    outline: none;
+    color: #ffffffca;
```
usa rgba

jdomingu98 commented on src/components/common/forms/form-field/form-field.js:
```diff
@@ -0,0 +1,58 @@
+        if(input.type === 'password')
+            input.type = 'text';
+        else
+            input.type = 'password';
```
Usa un ternario aqui

jdomingu98 commented on src/components/svg/close-modal.js:
```diff
@@ -0,0 +1,20 @@
+
+        return `
+            <button class="btn btn-primary p-0 text-light d-flex mt-4 ms-4" style="background-color: transparent; border:none;">
+                <svg xmlns="http://www.w3.org/2000/svg" width="30" height="30" fill="currentColor" class="bi bi-x-lg" viewBox="0 0 16 16">
```
y este svg? Si es un icono busca en bootstrap el equivalente

jdomingu98 commented on src/i18n/en.json:
```diff
@@ -38,7 +38,21 @@
+    "SPAN":{
+      "TERMS_CONDITIONS": "Terms and Conditions",
+      "ACCEPT":"I accept the ",
+      "OR": "- O R-"
```
Fallo con los espacios

jdomingu98 commented on src/i18n/en.json:
```diff
@@ -38,7 +38,21 @@
+    },
+    "SPAN":{
+      "TERMS_CONDITIONS": "Terms and Conditions",
+      "ACCEPT":"I accept the ",
```
Este y el de arriba los puedes juntar perfectamente y usar luego ${...}

jdomingu98 commented on src/main.js:
```diff
@@ -14,4 +14,22 @@ import style from './styles.css?inline';
+                { path: '/reset-password/:token', component: 'update-password' },
+                { path: '/privacy-policy', component: 'privacy-policy' },
+                { path: '/legal-notice', component: 'legal-notice' },
+                { path: '/terms-conditions', component: 'terms-conditions' },
```
Añade el path del ranking

jdomingu98 reviewed (changes requested):
He revisado un poco por encima y he parado porque sino no termino. Cambia lo que te pongo primero y luego te sigo diciendo.

Si ves cosas similares a las que te he comentado cambialas del tiron para lo alargar esto.

Otras cosas a cambiar:
- Añade algo de padding al icono del ojo de las contraseñas.
- Mensajes de errores algo mas descriptivos. No se lo que es un "Invalid password" cuando me registro
- Consistencia en los errores. De poco e sirve que no me coincidan las contraseñas si luego tambien me dice que es invalida.
- El icono de warning de los mensajes de errores cambialo por uno de error. Además, usa un icono de bootstrap, no el emoji en un span.
- Volviendo al icono del ojo, que cambie cuando habilitas para ver y para no ver

jdomingu98 commented:
La "X" del formulario sigue si estar fija

crist6494 committed:
refactor(close-modal): change svg to icon

crist6494 committed:
refactor(register-section): change msg and improve sintax

crist6494 committed:
styles(form-field): change color to rgba

crist6494 committed:
refactor(error-alert): change emoji to icon

crist6494 committed:
refactor(forms-field): add more padding and eye password

crist6494 committed:
styles(register-section): add position-fixed to icon-close

crist6494 committed:
refactor(modal-card): resolve wrong sintax

crist6494 committed:
feat(translator): add new translations

crist6494 committed:
refactor: resolve syntax class and attributes name

crist6494 committed:
styles(color): set global color

crist6494 committed:
feat(regex): add const regex variable

crist6494 committed:
fix(form-input): resolve space for long error msgs

crist6494 committed:
refactor(modal): remove esc function

jdomingu98 committed:
fix(modal): move modal logic to LandingPage

Also added functionality on two more buttons of LandingPage

jdomingu98 committed:
Merge branch 'betta' into feature/TR-39/Crear-modal-registro

jdomingu98 committed:
feat(AuthService): add login and forgot password endpoints links

jdomingu98 committed:
refactor(update-password): renamed to match route

jdomingu98 committed:
fix(components): deleted duplicated components

Snackbar is in the library as a service and the spinner one is duplicated in common components

jdomingu98 committed:
fix(pages index): update link of reset password

jdomingu98 committed:
fix(duplicated components): remove links

jdomingu98 committed:
style: simplify css and js behaviour

Changes mostly respecting to landing-auth-modal

jdomingu98 committed:
refactor(auth-modal): Changes directory name

jdomingu98 committed:
feat(landing-auth-modal): add new version of auth-modal

jdomingu98 committed:
style: fix identation

jdomingu98 committed:
feat(input-fields): add transition to errors in validation

jdomingu98 committed:
feat(signin): add connection to backend

jdomingu98 committed:
feat(otp): add template of otp modal

jdomingu98 committed:
style(landing-auth-modal): clean up js

jdomingu98 committed:
fix(forgot-password): make connection to backend

Resolve bugs related to the call of endpoint

jdomingu98 committed:
fix(forgot-password): print message error when method fails

jdomingu98 committed:
fix(login): add missing imports

Also add logout functionality

jdomingu98 committed:
refactor(auth): Delete UserService and change endpoint name

jdomingu98 committed:
fix: add missing imports, closing tags and edit transitions duration

jdomingu98 committed:
feat(signin-form): finish applying logic

jdomingu98 committed:
feat: add logic for 42 oauth

jdomingu98 committed:
refactor: added some minor changes

jdomingu98 committed:
feat(register-form): add almost all logic of registration process

jdomingu98 committed:
style(Snackbar): add div container

jdomingu98 committed:
feat(landing-otp-modal): Add otp modal and its events

jdomingu98 committed:
style(landing-auth-modal): increment ::backdrop shadow

jdomingu98 committed:
feat(i18n): add otp translations

jdomingu98 committed:
feat(landing-otp-modal): add username variable to match specification

jdomingu98 committed:
fix(username data): send username field value throught function

jdomingu98 committed:
fix(landing-otp-modal): NavigatorService incorrect name

jdomingu98 committed:
feat(login-form): add Toast when a OTP is required

jdomingu98 committed:
Merge pull request #25 from jdomingu98/feature/TR-118/Crear-modal-de-insercion-de-OTP

Modal OTP

jdomingu98 committed:
refactor(landing): move subcomponents to LandingPage

Because of the future merge of other branch that has a massive refactor

jdomingu98 committed:
style(Landing): simplify subscribe function

jdomingu98 committed:
refactor: delete older auth-modal components

register-section.js is still in the project because maybe i have to consult some logic

jdomingu98 committed:
refactor(landing components): simplify project structure

jdomingu98 committed:
fix(AuthService): change register api route and order endpoints

jdomingu98 committed:
feat(NavigatorService): add new route

jdomingu98 committed:
style(register-form): delete styles for checkbox

jdomingu98 committed:
feat(register-form): finish register validation

jdomingu98 committed:
refactor(register-section): delete unnecessary file

jdomingu98 committed:
refactor(auth forms): delete unnecessary message

jdomingu98 committed:
feat(i18n): add translations

jdomingu98 committed:
fix(i18n): add missing translations

jdomingu98 committed:
feat(landing components): apply translations

jdomingu98 committed:
style(terms link): add style to register terms link

jdomingu98 committed:
refactor(login-form): call NavigatorService instead of doing it manually

jdomingu98 commented:
Modal hecho.

Cosas faltantes por parte del backend:
- Fixear el endpoint del registro (ya hay una task creada)
- Terminar la confirmacion del registro
- Relacionado con lo primero, mandar literales de traduccion de los endpoint de auth faltantes

Del front falta añadir:
- Patch para confirmacion de cuenta (pendiente del back)
- Añadir literales a los archivos de traduccion (pendiente del back)
- Mergear la rama de optimizacion a betta. (resolver conflictos)

jdomingu98 reviewed (approved):
La he hecho yo, asi que obviamente lo veo bien quitando lo comentado que no depende de mi

crist6494 commented:
Lo he estado probando y alguna cosas no cuadran: 
-Se puede registrar sin aceptar la casilla de términos y condiciones
-A veces la casilla de términos no funciona correctamente y no se marca o se desmarca
-si el el registro es mayúscula no puedo tener nombre en minúscula? esto es pregunta, porque mis usurios con. nombre Cristian no puedo acceder

-.input:-webkit-autofill {
    -webkit-box-shadow: 0 0 0 1000px transparent inset !important;
    -webkit-text-fill-color: rgba(255, 255, 255, 0.79) !important;
    background-color: transparent !important;
} con esto cuando haces autocompletar se matiene el color dentro del input

-Si me sale un error en rojo en un input y me cambio al register o login y me vuelvo a cambiar se matiene el error  

El resto lo veo bien

jdomingu98 commented:
@crist6494 te contesto así porque no puedo responderte debajo de lo que has puesto:

- No se puede registrar sin pulsar puesto que tiene el campo required en el input. Lo que ves es un bug visual. Si te pasa sin darle siquiera al checkbox le echare un vistazo.
- Lo que pasa con el checkbox es que esta algo bug y no se el motivo. Cuando pulsas en el div va perfecto, pero en el propio checkbox y en el label hace una especie de parpadeo raro.
- El registro no es en mayuscula. Osea, el dato te lo coge bien lo que pasa que se me ha olvidado borrar el estilo.
- No veia necesario lo del autofill la verdad, pero ya que lo has puesto le echo un vistazo
- Cuando escribes o cierras se resetea.

jdomingu98 committed:
refactor(landing forms): delete uppercase text on inputs

crist6494 commented:
Te escribo asi entonces @jdomingu98 xd, vale pues si todavia esta el componente que hice del checbox úsalo si quieres, lo de la máyuscula eso cambiarlo, y yo lo que me refiero  es que si escribes algo en un input o algun mensaje de error, el del  registro por ejemplo y luego me cambio al login y después vuelvo al registro debería estar reiniciado no?, es que ahora mismo se mantiene lo que dejaste escrito.

jdomingu98 committed:
style(inputs): add autofill styles

jdomingu98 commented:
@crist6494 he visto tu componente del checkbox y es lo mismo que tengo puesto. Osea un input y un label, nada más. El motivo por el que esta bug es otra cosa

jdomingu98 commented:
Lo del formulario que no se borra cuando cambias es una feature. Imaginate que le das sin querer a uno de los botones que te hacen animacion y se te borra lo rellenado

crist6494 commented:
@jdomingu98 a vale, de todos modos pondría ese q así se diferencia algo y no tener el checkbox por defecto, pero lo q veas

jdomingu98 committed:
refactor(landing components): delete more unnecessary files

jdomingu98 committed:
fix(dialog events): edit some keydown events

jdomingu98 committed:
style(hex colors): change to global variables

jdomingu98 committed:
fix(checkbox): now has better performance

In chrome its looks fine, in opera is still buggy

jdomingu98 commented:
@crist6494 revisa de nuevo que ya hice los cambios

jdomingu98 committed:
refactor(landing): solve conflicts

jdomingu98 committed:
fix(ResetPassword): change attribute name and simplify

jdomingu98 committed:
refactor(const): delete unnecessary variables

ruzafa8 reviewed (approved):
gualililuti
